src/app/lib/seoUtils/getListItems/index.ts

```diff
@@ -0,0 +1,19 @@
+import { CurationData } from '../../../models/types/curationData';
+
+export default (curations: CurationData[]) => {
```

Comment:
Do `getListItems` and `getItemList` need to be separate things? Can the one function not just return the data needed? Just confused me a little with the naming, and I can't see them being used independently of one another.

Reply:
Good shout, they technically could be reused later, but this is perhaps premature optimisation. I will combine them into a single function.